lib/matplotlib/backend_tools.py

```diff
@@ -1020,6 +1019,35 @@ def _mouse_move(self, event):
         self.toolmanager.canvas.draw_idle()
 
 
+class ToolHelpBase(ToolBase):
+    description = 'Print tool list, shortcuts and description'
+    default_keymap = rcParams['keymap.help']
+    image = 'help.png'
+
+    def _get_help_text(self):
+        entries = []
+        for name, tool in sorted(self.toolmanager.tools.items()):
+            if not tool.description:
+                continue
+            entries.append(
+                "{}: {}\n\t{}".format(
+                    name,
+                    ", ".join(self.toolmanager.get_tool_keymap(name)),
+                    tool.description))
+        return "\n".join(entries)
+
+    def _get_help_html(self):
+        fmt = "<tr><td>{}</td><td>{}</td><td>{}</td></tr>"
+        rows = [fmt.format("NAME", "KEYS", "DESCRIPTION")]
```

Comment:
I'd use bold instead of uppercase, e.g. `"<b>Name</b>"` instead of `"NAME"`

Reply:
Agreed that it looks better.  Changed accordingly.